### DISCUSSION TITLE
Rocket Chat Integration with janssen server

### DISCUSSION THREAD

**Discussion by 0786ShivaniBansal** [JanssenProject/jans#5765] [Q&A]
Hii Team we are doing OIDC Integration Janssen server with the Rocket Chat application we are going through every step but we are not able to be redirected to the rocket chat dashboard getting back to the login.htm of rocket chat application instead of rocket chat dashboard I'm attaching configuration screenshots of both the Janssen please verify.
This is the documentation we are following [https://www.gluu.org/docs/gluu-server/4.3/integration/saas/rocketchat/#configuring-rocket-chat]
![MicrosoftTeams-image (19)](https://github.com/JanssenProject/jans/assets/95275127/5b16a4db-eb89-4283-a7dc-b61b23defcfd)
 server and rocket chat application
![MicrosoftTeams-image (20)](https://github.com/JanssenProject/jans/assets/95275127/53193da2-3621-49e6-927d-f16fa52e9345)
![MicrosoftTeams-image (21)](https://github.com/JanssenProject/jans/assets/95275127/cbe6d30e-6039-4e18-b474-207cda3c60cc)
![MicrosoftTeams-image (22)](https://github.com/JanssenProject/jans/assets/95275127/7c7322a8-d645-4b27-b42e-6a40cdc9c5d6)

**Comment by mmrraju** [comment]
hello @0786ShivaniBansal

I just checked at my end get same issue. Let me find out where is the problem.

**Reply by 0786ShivaniBansal** [reply]
Hii @mmrraju did you get a chance to look into waiting for an update, thanks.

**Reply by mmrraju** [reply]
Hello @0786ShivaniBansal, sorry for late, I'v analysis with all `logs` file specially `jans-auth.log`. So for the `rocket-chat` I observe that `/jans-auth/restv1/userinfo` endpoint is not being called by rocket-chat `oauth` app. **That is the main reason.**

bellow request is missing when we try to integrate with rocket-chat. 
```
2023-08-04 14:13:45,619 TRACE [qtp283318938-18] c96f49d8-369f-48c3-bb6d-995d5641c881 [io.jans.as.server.auth.AuthenticationFilter] (AuthenticationFilter.java:150) - Get request to: 'https://raju.jans16.me/jans-auth/restv1/userinfo'

```

I test with another SSO flow to confirm that from our server end, is `/jans-auth/restv1/userinfo` endpoint working properly? yes it's working.

**Reply by 0786ShivaniBansal** [reply]
Hii @mmrraju thanks for the update can you please suggest some free custom oidc applications to integrate with the Janssen server

**Comment by 0786ShivaniBansal** [comment]
Hii Team waiting for an update please reply as soon as possible thanks

**Comment by nynymike** [comment]
This is community support. Maybe as part of the community you can research the issue and post the solution here ;-)
